zou/app/blueprints/crud/person.py

```diff
@@ -22,11 +23,11 @@ def all_entries(self, query=None, relations=False):
 
         if permissions.has_manager_permissions():
             if request.args.get("with_pass_hash") == "true":
-                return [person.serialize() for person in query.all()]
+                return [person.serialize(relations=True) for person in query.all()]
             else:
-                return [person.serialize_safe() for person in query.all()]
+                return [person.serialize_safe(relations=True) for person in query.all()]
```

Comment:
Can you make lines with length < 80 chars? I'm not sure that this one is not too wide.